cumulusci/tasks/bulkdata/generate_mapping.py

```diff
@@ -161,36 +168,32 @@ def _build_mapping(self):
         objs = set(self.schema.keys())
         stack = self._split_dependencies(objs, self.refs)
 
-        field_sort = (
-            lambda f: "  " + f
-            if f == "Id"
-            else (" " + f if f in self.core_fields else f)
-        )
+        def strip_namespace(element):
+            ns = self.project_config.project__package__namespace
+            if self.options["strip_namespace"] and ns and element.startswith(ns):
+                return element[len(ns) + 2 :]
+            else:
+                return element
 
         self.mapping = {}
         for obj in stack:
-            key = f"Insert {obj}"
+            key = f"Insert {strip_namespace(obj)}"
             self.mapping[key] = {}
-            self.mapping[key]["sf_object"] = f"{obj}"
-            self.mapping[key]["table"] = f"{obj}"
+            self.mapping[key]["sf_object"] = strip_namespace(obj)
```

Comment:
Call strip_namespace once and assign it to something so it doesn't need to be called twice